packages/breadboard-web/src/boards/open-alex-entity-search-list.ts

```diff
@@ -4,177 +4,108 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { Schema, base, board, code } from "@google-labs/breadboard";
-import { core } from "@google-labs/core-kit";
+import {
+  board,
+  enumeration,
+  input,
+  object,
+  output,
+} from "@breadboard-ai/build";
+import { fetch, code } from "@google-labs/core-kit";
 
-import { templates } from "@google-labs/template-kit";
+import { urlTemplate } from "@google-labs/template-kit";
 
-const spread = code<{ object: object }>((inputs) => {
-  const object = inputs.object;
-  if (typeof object !== "object") {
-    throw new Error(`object is of type ${typeof object} not object`);
-  }
-  return { ...object };
-});
-
-const openAlexEntities: Schema = {
+const entity = input({
   description: "Entity type to search for",
-  type: "string",
-  default: "works",
-  enum: [
-    "works",
+  type: enumeration("works",
     "authors",
     "sources",
     "institutions",
     "topics",
     "publishers",
     "funders",
-    "concepts",
-  ],
+    "concepts"),
+  default: "works",
   title: "Entity",
-};
-
-const pageInputSchema: Schema = {
-  type: "integer",
-  default: "1",
-  title: "Page number",
-  description: "Page number to return",
-};
+});
 
-const perPageInputSchema: Schema = {
-  type: "integer",
+const page = input({
```

Comment:
This should have `type: "number"`

Reply:
I believe this will require a change to the `urlTemplate` node, as the wildcard type is currently set to "string". I'll be putting in a new PR for this soon, but for now I've added in TODO comments for this to be changed to "number" once the node has been updated.